### PR TITLE
002 branding

### PR DESCRIPTION
**Pull request by Katiemcallister**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
```
Comment (shaunganley) [inline .vscode/launch.json]: doesn't look right for frontend URL

File: src/app.ts [hunk 2]
```diff
@@ -3,15 +3,28 @@ import nunjucks from "nunjucks";
...
+import { dateFilter } from "./filter/DateFilter";
+
 
 const app = express();
 
+
 nunjucks.configure('views', {
     autoescape: true,
     express: app
 });
```
Comment (shaunganley) [inline src/app.ts]: think you can remove this as you've added the part below

File: src/app.ts [hunk 3]
```diff
@@ -29,4 +42,5 @@ app.listen(3000, () => {
     console.log('Server started on port 3000');
 });
 
-app.get('/', getAllDatabases);
\ No newline at end of file
+app.get('/homepage' , getHomePage)
+app.get('/jobRoles', getAllJobRoles);
```
Comment (shaunganley) [inline src/app.ts:40]: by adding all of the job role stuff in this branch we can't progress the branding ticket until the job roles ticket is complete. I recommend removing all the job roles stuff and just make this PR about the branding

File: views/homepage.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+
+    <body>
+        <nav class="header">
```
Comment (shaunganley) [inline views/homepage.html]: move this section to a header.html file and import it here so we can reuse on all pages

File: views/homepage.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                </div>
+             
+            </div>
+            </nav>
+
+
+            <img src="https://cdn.phenompeople.com/CareerConnectResources/KAINGB/images/End-User-Size-1-1643385498708.jpg" alt="homepageBackgroundImage" style="max-width: 100%; height: auto;" >
+
+        <!-- Footer Start -->
+                            <div class="footer-basic">
```
Comment (shaunganley) [inline views/homepage.html]: move this section to a footer.html file and import it here so we can reuse on all pages

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Review by BenjaminBrashaw — APPROVED** [review]

**Pull request merged**